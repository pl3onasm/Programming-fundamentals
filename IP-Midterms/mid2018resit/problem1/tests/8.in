4
1 2 4 8 5 6
2 6 5 4 6 7
3 1 3 5 4 3
3 9 4 7 5 9
