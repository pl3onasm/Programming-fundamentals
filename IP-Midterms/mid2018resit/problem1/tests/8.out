1 1 6 9
